src/components/Photo/Selfie.js

```diff
@@ -37,6 +39,7 @@ export default class Selfie extends Component<Props, State> {
     hasBecomeInactive: false,
     hasCameraError: false,
     snapshotBuffer: [],
+    btnDisabled: false
```

Comment:
I personally think that naming could be better, e.g. `isCapturing`, that would represent the state rather than specific component prop. In such case you disable button with `disabled={isCapturing || hasCameraError}`. Also, in such case this would make sense when you would set `this.setState({ isCapturing: true })` inside "takeSelfie" method, and `this.setState({ isCapturing: false })` inside "handleSelfie" - so it's a start and finish methods.